docs/constants/FeatureFlags.js

```diff
@@ -1,3 +1,14 @@
+// If you change the flag value, you need to restart the dev server.
+const flags = {
+  isEasInFeaturePreview: true,
```

Comment:
Should we maybe remove this rather than renaming?

Reply:
we might want to keep it around in the future, eg: for eas update